app/services/idv/steps/verify_base_step.rb

```diff
@@ -4,7 +4,7 @@ class VerifyBaseStep < DocAuthBaseStep
       AAMVA_SUPPORTED_JURISDICTIONS = %w[
         AR AZ CO CT DC DE FL GA IA ID IL IN KY MA MD ME MI MO MS MT ND NE NJ NM
         PA RI SC SD TX VA VT WA WI WY
-      ].freeze
+      ].to_set.freeze
```

Comment:
I still have trouble recognizing when we do or don't need to `require` the implementation. Is `Set` always available? Should we err on the safe side and always `require 'set'` anywhere we're expecting to use it?

Reply:
I think in current versions of ruby, it's in the default load:

```
irb
irb(main):001:0> require 'set'
=> false
```
(require returns `true` when something is actually loaded, and `false` when it's already been loaded)